subsys/bluetooth/shell/bt.c

```diff
@@ -2179,6 +2179,13 @@ static int cmd_conn_phy_update(const struct shell *shell, size_t argc,
 	struct bt_conn_le_phy_param param;
 	int err;
 
+	if (default_conn == NULL) {
+		z_shell_fprintf(shell, SHELL_ERROR,
+				"%s: at least, one connection is required\n",
+				shell->ctx->active_cmd.syntax);
+		return -EINVAL;
```

Comment:
I'm not sure of the importance, but it seems that when bad arguments are sent to shell commands, we return `-ENOEXEC`. 

So I suggest we return that here as well.

Reply:
If we have a common way to handle this bad argument return, I will follow it.

Reply:
It is not allowed to use function `z_shell_fprintf` inside the command handler. It is an internal function used by the shell.
You can implement it either:
```
shell_fprintf(shell, SHELL_ERROR,
				"%s: at least, one connection is required\n",
				shell->ctx->active_cmd.syntax);
```
or
```
shell_error(shell,  "%s: at least, one connection is required",
			    shell->ctx->active_cmd.syntax);
```